sdk/formrecognizer/azure-ai-formrecognizer/MIGRATION_GUIDE.md

```diff
@@ -636,10 +636,11 @@ for doc in model.training_documents:
 ```
 
 Train a custom model with `3.2.x`:
+***As of 3.2.0b3, `begin_build_model()` has a required `build_mode` parameter. For more information about build modes, see: https://aka.ms/azsdk/formrecognizer/buildmode. ***
 ```python
 document_model_admin_client = DocumentModelAdministrationClient(endpoint, AzureKeyCredential(key))
 poller = document_model_admin_client.begin_build_model(
-    container_sas_url, model_id="my-model-id", description="my model description"
+    container_sas_url, "template", model_id="my-model-id", description="my model description"
```

Comment:
it seems to be rendering wrong for some reason. Maybe because of the space between the sentence and ***?

Reply:
I needed to add the space or else the *** seemed to be added to the url. :/ I'll reword so we don't have this problem.